irrep/_version.py

```diff
@@ -1 +1 @@
-__version__="1.5.2"
```

Comment:
I moved _bersion.py to __version.py  so, the old file shopuld be removed from the tree.

Reply:
Oh, I see. I will change it.